.github/workflows/compile-tests.yml

```diff
@@ -27,6 +27,7 @@ jobs:
       - name: Check prettier codestyle
         working-directory: tests/assemblyscript
         run: |
+          npm install prettier
```

Comment:
Thanks a lot for the change. I see prettier is already part of the package.json, so maybe we could run `npm install` command before calling `prettier-format-check` (e.g. have another step with `npm install`) instead of doing it in the `Build tests` step?